### DISCUSSION TITLE
send a mail the day of the event

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#152] [Q&A]
Hi,
I would like to send an email to the organiser of an event once the event is finished. Is there a hook that could help me

**Comment by liedekef** [comment]
The next version has extra hooks that run once a day. From the changelog:
```
 Added actions eme_events_daily_action, eme_members_daily_action and eme_daily_action, which all 3 run on a daily basis without arguments
 Since all 3 run on a daily basis, you could decice to put everything in eme_daily_action, but the distinction might be usefull.
```
Then you can hook into those :-)

**Comment by f14m07** [comment]
Hi Franky,
Thanks, I'll wait for the next version. One point worries me, since there is no parameter, will the hook automatically process the day's events?
